comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -133,6 +133,7 @@
       <packagereq type="default">tfm-rubygem-puppetdb_foreman</packagereq>
       <packagereq type="default">tfm-rubygem-radcli</packagereq>
       <packagereq type="default">tfm-rubygem-rainbow</packagereq>
+      <packagereq type="default">tfm-rubygem-rbnacl</packagereq>
```

Comment:
this also needs to go to `comps-foreman-plugins-el8.xml` without the tfm prefix

Reply:
changed